modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/BoundObjectDefinitionsExportImportTest.java

```diff
@@ -57,6 +57,11 @@ public void testExportImportBoundObjectDefinitions() throws Exception {
 			"test-bound-object-definitions.json", null,
 			"TestObjectDefinition1");
 
+		testExportImport(
+			"test-bound-object-definitions.published.json",
+			"test-bound-object-definitions.json", null,
+			"TestObjectDefinition1");
+
```

Comment:
Since you are differentiating here (using.published), it is better to specify in the previous test.
So it could be something like this:
```
testExportImport(
	"test-bound-object-definitions.**draft**.json",
	"test-bound-object-definitions.**draft**.json", null,
	"TestObjectDefinition1");
```
what do you think?